src/jv_aux.c

```diff
@@ -538,6 +602,47 @@ static int string_cmp(const void* pa, const void* pb){
   return r;
 }
 
+jv jv_paths(jv input){
```

Comment:
Can we have a comment on what this is supposed to do?  Is it output an array of paths to scalar values?